cmd/finch/nerdctl_test.go

```diff
@@ -351,32 +351,6 @@ func TestNerdctlCommand_run(t *testing.T) {
 				ncsd.EXPECT().LookupEnv("FINCH_DOCKER_COMPAT").Return("", false)
 			},
 		},
-		{
-			name:    "with --add-host flag and special IP by space",
-			cmdName: "run",
-			args:    []string{"--rm", "--add-host", "name:host-gateway", "alpine:latest"},
-			wantErr: nil,
-			mockSvc: func(
-				t *testing.T,
-				lcc *mocks.LimaCmdCreator,
-				ncsd *mocks.NerdctlCommandSystemDeps,
-				logger *mocks.Logger,
-				ctrl *gomock.Controller,
-				fs afero.Fs,
-			) {
-				getVMStatusC := mocks.NewCommand(ctrl)
-				lcc.EXPECT().CreateWithoutStdio("ls", "-f", "{{.Status}}", limaInstanceName).Return(getVMStatusC)
-				getVMStatusC.EXPECT().Output().Return([]byte("Running"), nil)
-				logger.EXPECT().Debugf("Status of virtual machine: %s", "Running")
-				ncsd.EXPECT().LookupEnv("FINCH_DOCKER_COMPAT").Return("", false)
-				logger.EXPECT().Debugf(`Resolving special IP "host-gateway" to %q for host %q`, "192.168.5.2", "name")
-				ncsd.EXPECT().LookupEnv("COSIGN_PASSWORD").Return("", false)
-				c := mocks.NewCommand(ctrl)
-				lcc.EXPECT().Create("shell", limaInstanceName, "sudo", "-E", nerdctlCmdName, "run",
-					"--rm", "--add-host", "name:192.168.5.2", "alpine:latest").Return(c)
-				c.EXPECT().Run()
-			},
-		},
 		{
 			name:    "with --add-host flag but without using special IP by space",
```

Comment:
Do we need these two?

Reply:
I delete those two

Reply:
"with --add-host flag but without using special IP by space" is still in the PR.